org/org.yaml

```diff
@@ -828,3 +828,58 @@ orgs:
         privacy: closed
         repos:
           resolution: maintain
+  tektoncd-catalog:
+    admins:
+    - wlynch
+    - bobcatfish
+    - tekton-robot
+    - thelinuxfoundation
+    - vdemeester
+    - afrittoli
+    - dibyom
+    - QuanZhang-William
+    - vinamra28
+    billing_email: info@cd.foundation
+    company: ""
+    default_repository_permission: none
+    description: ""
+    email: ""
+    has_organization_projects: true
+    has_repository_projects: true
+    location: ""
+    name: ""
+    members:
+    - divyansh42
+    - iancoffey
+    - jerop
+    - jjasghar
+    - piyush-garg
+    - priyawadhwa
+    - PuneetPunamiya
+    - sm43
+    teams:
+      golang.maintainers:
```

Comment:
What about other repos,, will you add them in a later PR?

Reply:
Heh, I guess the remaining teams will be added in a follow-up PR

Reply:
Hi @afrittoli. Yeah, you are right. I'm currently just migrating golang repo and will use this repo as a test.

Will create other teams in 1 PR after validating this is correct.